src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -94,8 +94,9 @@ start_metadata_sync_to_node(PG_FUNCTION_ARGS)
 	{
 		ereport(ERROR, (errcode(ERRCODE_OBJECT_NOT_IN_PREREQUISITE_STATE),
 						errmsg("you cannot sync metadata to a non-existent node"),
-						errhint("First, add the node with SELECT master_add_node(%s,%d)",
-								nodeNameString, nodePort)));
+						errhint(
+							"First, add the node with SELECT master_add_node(\'%s\',%d)",
```

Comment:
Instead of manually wrapping %s with single quotation marks, you can use quote_literal_cstr function